### PR TITLE
[doc] rephrase actor handles msgs of common super type

### PR DESCRIPTION
**Pull request by robstoll**
## References

References #xxxx

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/actors.md [hunk 1]
```diff
@@ -80,9 +80,8 @@ the next behavior is "the same as the current one".
 The type of the messages handled by this behavior is declared to be of class
 `Greet`@java[.]@scala[, meaning that `message` argument is also typed as such.
 This is why we can access the `whom` and `replyTo` members without needing to use a pattern match.]
-Typically, an actor handles more than one specific message type and then there
-is one common @scala[`trait`]@java[`interface`] that all messages that the
-actor can handle @scala[`extends`]@java[`implements`].
+Typically, an actor handles more than one specific message type where all of them
+@scala[`extends`]@java[`implements`] directly or indirectly the same common @scala[`trait`]@java[`interface`].
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/actors.md]:
```suggestion
directly or indirectly @scala[`extend`]@java[`implement`] a common @scala[`trait`]@java[`interface`].
```

**Review by raboof — COMMENTED** [review]
Suggested a further tweak

**Review by johanandren — APPROVED** [review]
Looks like an improvement to me.

**Comment by robstoll** [comment]
@raboof I would not render `extend` as code as it is not a keyword anymore but up to you

**Review by raboof — APPROVED** [review]

**Pull request merged**